core/input.js

```diff
@@ -329,4 +317,19 @@ class Input {
   }
 }
 
+/**
+ * Enum for alignment of inputs.
+ * @enum {number}
+ * @alias Blockly.Input.Align
+ */
+const Align = {
+  LEFT: -1,
+  CENTRE: 0,
+  RIGHT: 1,
+};
+exports.Align = Align;
+
+// Add Align to Input so that `Blockly.Input.Align` is publicly accessible.
+Input.Align = Align;
```

Comment:
Could we instead just have `Input.Align = {Left: -1 ...}`? It seems cleaner to me to only have the one way of using the enum and AFAICT we are already using pattern in some places. (`ContextMenuRegistry.ScopeType`, `ASTNodes.type`)

Reply:
There is a long thread about this in the chat and Christopher voted for this way so that, hypothetically, if you only need to reference the enum you can import just the enum instead of the whole class. In practice, that only applies to people doing advanced compilation for now but if we got rid of blockly.js in the future and moved to direct imports then someone could import just the enum they need (this is likely for the NameType enum that they don't actually need to import Names, not sure about the Input one but seems plausible).

I'm happy to change it but you'd have to argue with Christopher about it. :D

Reply:
Hmm since we are doing it in quite a few other places it seems like we would have to go back in the future anyway and audit this if the above situation does end up happening. But totally appreciate that I am stepping into a larger conversation so happy for this to go in as is : )